MinecraftClient/Mapping/World.cs

```diff
@@ -16,11 +16,22 @@ public class World
         /// </summary>
         private Dictionary<int, Dictionary<int, ChunkColumn>> chunks = new Dictionary<int, Dictionary<int, ChunkColumn>>();
 
+        /// <summary>
+        /// The dimension info of the world
+        /// </summary>
+        private Dimension dimension;
```

Comment:
How about initializing this with a placeholder Dimension? Would save checking for `null` every time?
_(Note, you can save defining getter and setter using `public Dimension dimension { get; set; }`)_